module/VuFind/src/VuFind/Search/Favorites/Options.php

```diff
@@ -55,6 +55,7 @@ public function __construct(\VuFind\Config\PluginManager $configLoader)
         $this->sortOptions = [
             'title' => 'sort_title', 'author' => 'sort_author',
             'year DESC' => 'sort_year', 'year' => 'sort_year asc',
+            'last_saved DESC' => 'sort_saved', 'last_saved' => 'sort_saved asc',
```

Comment:
I wonder if we should make the sort options configurable. Not necessary for this PR since you're just following the existing precedent, but maybe useful as a follow-up. If you're not interested in doing it, I can put it on my own to-do list; shouldn't take too much effort -- we probably just need to create a [List_Sorting] section in config.ini near the [Social] section.

Reply:
Yes, I also though about this. I should be able to do it.